app-modules/notification/database/factories/OutboundDeliverableFactory.php

```diff
@@ -46,6 +48,16 @@ class OutboundDeliverableFactory extends Factory
     public function definition(): array
     {
         return [
+            'channel' => fake()->randomElement(NotificationChannel::cases()),
+            // TODO Introduce a real fake test notification for this purpose
+            'notification_class' => 'test_notification',
```

Comment:
Let's create a ticket for this, point it with what you think it would take, and add as much detail as you can. Including the file location here that this would need to be changed.

Then let's remove this TODO.

Reply:
I went ahead and addressed this in the PR, as the test classes already existed.